main.py

```diff
@@ -95,6 +95,7 @@ def execute_consistency(self):
                 circuit.is_enabled()
                 and not circuit.is_active()
                 and not circuit.lock.locked()
+                and not circuit.recent_updated()
```

Comment:
Also `check_traces` are still being called per EVC here, so you're not benefiting much from the bulk traces endpoint that you've implemented. Once you have all the EVCs that haven't had recent changes and aren't active and enabled, you can perform the bulk trace for all pairs of UNIs of all of these EVCs.

Reply:
@gretelliz alternatively, a simpler and intermediary approach would be to store `flow_removed_at` and filter only in memory using `self.circuits` since mef_eline still has loaded circuits in memory and hasn't completely offloaded to the DB, and then when more things are offloaded we can move this too in the future. Iterating over the EVCs in memory will typically be a collection with < 500 ish elements so that'd be OK. The code will look similar to what you have here except that `recent_updated` won't hit the DB and will just compare datetime. You can still export `flow_removed_at` on `as_dict` and in the models to be stored in the DB, but it only write to the DB in places where it already upserts.